components/switch/test/switch.visual-diff.js

```diff
@@ -27,8 +37,20 @@ describe('d2l-switch', () => {
 			{ name: 'off', selector: '#off' },
 			{ name: 'off-focus', selector: '#off', action: (selector) => page.$eval(selector, (elem) => elem.focus()) },
```

Comment:
I think you'll need to use our `forceFocusVisible` helper -- have a look at what `button-icon` is doing. Programatically calling `.focus()` doesn't trigger its heuristic, which requires a combination of keyboard events for it to believe that the focus merits the visible treatment.

Reply:
Nice catch, I didn't double check that the focus tests were still working (it looks like they aren't). I'll take a look at the forceFocusVisible helper